### PR TITLE
Add tendrl-api firewall configuration

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Signed-off-by: Timothy Asir J <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@r0h4n , @nthomas-redhat , @shtripat , @mbukatov , Please review

**Comment by mbukatov** [comment]
I have checked only https://github.com/Tendrl/monitoring-integration/pull/338 so far, but my comments there are general enough to be applicable here.

File: config/tendrl-api.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>tendrl-api</short>
+  <description>Enable Port for Tendrl API</description>
+    <port protocol="tcp" port="9292"/> <!-- api -->
+</service>
```
Comment (nthomas-redhat) [inline config/tendrl-api.xml]: Why this is kept in /config here which is different from monitoring-integration?

File: tendrl-api.spec [hunk 2]
```diff
@@ -102,6 +103,10 @@ getent passwd %{app_user} > /dev/null || \
 %post httpd
 setsebool -P httpd_can_network_connect 1
 systemctl enable tendrl-api >/dev/null 2>&1 || :
+firewall-cmd --reload >/dev/null 2>&1 || :
+
+%postun
+firewall-cmd --reload >/dev/null 2>&1 || :
```
Comment (nthomas-redhat) [inline tendrl-api.spec]: Why reload command is called in two places?

Comment (TimothyAsirJeyasing) [reply tendrl-api.spec]: one is after installation and another one is after uninstall

**Pull request merged**